apps/condo/domains/miniapp/schema/B2BApp.js

```diff
@@ -121,10 +121,12 @@ const B2BApp = new GQLListSchema('B2BApp', {
             const newItem = { ...existingItem, ...resolvedData }
             if (newItem.isGlobal) {
                 if (!newItem.appUrl) {
+                    // TODO(pahaz): use GQLError
```

Comment:
Are you sure you'll come back to this?

Reply:
Really not. I want to add some TODO for that. It's not a part of this PR because it's too big and I'm trying not to change match.